speech-commands/demo/ui.js

```diff
@@ -189,6 +193,12 @@ export function plotPredictions(canvas, candidateWords, probabilities, topK) {
     for (const word in candidateWordSpans) {
       if (word === topWord) {
         candidateWordSpans[word].classList.add('candidate-word-active');
+        if (timeToLiveMillis != null) {
+          setTimeout(
+              () => candidateWordSpans[word].classList.remove(
```

Comment:
optional: i suggest adding a nullcheck guard here, like "if (candidateWordSpans[word]) {..." in case this timeout logic races against the list of words changing. I realize it's unlikely, but GUI races are annoying to debug, and an unlucky future refactor could make this more probable in the future...

Reply:
Good point. Done. Thanks.